### ISSUE TITLE
send msg to contact via name in python

### ISSUE THREAD

**Issue by babaksit** [vysheng/tg#743]
hi.
is it possible to send msg via contact name in python like the lua send_msg function ?
if it is not possible , should i create a tgl.Peer object and assign every attribute of the object?
thanks.

**Comment by Mattze96** [comment]
Yeah, just make a peer list at the beginning after the contact_list_cb for example and then iterate later trough the obejcts by name.
Sent from my Cyanogen phone
Am 18.08.2015 08:01 schrieb babaksit <notifications@github.com>:hi.
is it possible to send msg via contact name in python like the lua send_msg function ?
if it is not possible , should i create a tgl.Peer object and assign every attribute of the object?
thanks.

**Comment by michelePap** [comment]
Hi, how want to send a message only to one user (myself) with the python script. How can i do?
